OWNERS

```diff
@@ -4,4 +4,9 @@ approvers:
   - jberkus
   - jeefy
   - mrbobbytables
+  - natalisucks
+  – nimbinatus
+  – reylejano
```

Comment:
@natalisucks It appears there are some new entries in the YAML with the character U+2013 "–" instead of the expected ASCII character U+002d "-". This might be due to some copy-paste encoding error _(even the [k8s-ci-bot ](https://github.com/kubernetes/contributor-site/pull/458/files#r1394778604) seems unhappy)_.  
I've suggested the changes with the correct symbol, hope it will make applying those suggestions easier.
```suggestion
  - nimbinatus
  - reylejano
```